content/dictionary/app.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: app
-definition: ''
-perspectives: []
+definition: App is short for application that is meant for end users.
+sources: 
+- sourceurl: https://en.wikipedia.org/wiki/Application_software
+perspectives: 
+- meaning: App is either used for mobile app or web app. It's software for clients, not integrated systems, internal machinery, etc.
```

Comment:
```suggestion
definition: App is short for application that is meant for end users. App is either used for mobile app or web app. It's software for clients, not integrated systems, internal machinery, etc.
sources: 
- sourceurl: https://en.wikipedia.org/wiki/Application_software
perspectives:
```
I like the extra information you added in the perspectives, but I think it might be better as part of the definition.